### ISSUE TITLE
What does the result of prediction of classification mean?

### ISSUE THREAD

**Issue by Scott534** [ibm-granite/granite-tsfm#115]
In the result of prediction which has 3 labels, I got an array of (n,3) in result[0][0], where n is the length of input data. What do the numbers mean?

e.g. I got array([3720297. , 3823655.2, 1119286.9], dtype=float32) in one row of the result.

By the way, I labeled the data with [-1,0,1], is it suggested that I should label the data only with numbers >=0?

**Comment by wgifford** [comment]
Hi @Scott534 the number of outputs should be equal to the number of classes you have in your input. The outputs are the raw unnormalized logits. If you take the softmax of these values, you can get to probabilities for each class.

The classification models are trained with cross entropy loss, which I believe is expecting integer labels starting at 0. https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html

**Comment by Scott534** [comment]
Understand, thanks for your help!
